README.md

```diff
@@ -64,10 +64,16 @@ Make sure you have the following prerequisites:
 
 First, get ExternalDNS:
 
+**To install all dependencies, make sure to install [dep](https://github.com/golang/dep) first.**
+
 ```console
-$ go get -u github.com/kubernetes-incubator/external-dns
+$ git clone https://github.com/kubernetes-incubator/external-dns.git && cd external-dns
```

Comment:
We should probably enter `$GOPATH` before.

Reply:
Why? Aren't we building go `1.9` and tip? If so, it shouldn't be necessary.

Reply:
Oh, I didn't know that.

Reply:
We can merge then?